ui/pages/extensions.js

```diff
@@ -0,0 +1,138 @@
+import { Grid, Typography, Button, Switch } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+import { useState } from "react";
+import FlipCard from "../components/FlipCard";
+
+const styles = (theme) => ({
+  card : {
+    height : '100%',
+    width : "40%",
+    backgroundColor : "transparent",
+    perspective : theme.spacing(125),
+    margin : theme.spacing(5),
+    overFlow : "hidden",
+  },
+  innerCard : {
+    padding : theme.spacing(2),
+    borderRadius : theme.spacing(1),
+    transformStyle : "preserve-3d",
+    boxShadow : "0 4px 8px 0 rgba(0,0,0,0.2)",
+    backgroundColor : "#fff",
+    cursor : "pointer",
+    transition : `transform 1ms`,
+    transformOrigin : "50% 50% 10%",
+
+  },
+  button : {
+    marginRight : "0.5rem",
+    borderRadius : 5,
+    minWidth : 100,
+    color : "#fff",
+    "&:hover" : {
+      boxShadow : "0px 2px 4px -1px rgb(0 0 0 / 20%), 0px 4px 5px 0px rgb(0 0 0 / 14%), 0px 1px 10px 0px rgb(0 0 0 / 12%)"
+    },
+  },
+  grid : {
+    margin : "1rem"
+  },
+  text : {
+    padding : "1rem"
+  },
+  img : {
+    marginLeft : "0.5rem",
+    height : 'auto',
+    width : "auto",
+    maxWidth : "100px",
+    maxHeight : "100px"
+  },
+  content : {
+    textAlign : "right",
```

Comment:
```suggestion
    textAlign : "left",
```

Reply:
This won't match the desired Figma design, It will bring the Text above the icon.